### DISCUSSION TITLE
How soon before China release a 'chia plotting' asic farmer (miner)

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#5254] [General]
I find that plotting is every bit as ASIC 'complex' as BTC mining, its the fastest CPUS with the most cores, and the largest ram disks ( 1TB on board DDR4 )  that make the fastest plots that win this game, how long before china has a CHIA box that spit's out 108GB plots? (100/day) I think very soon

Seriously for all the talk about 'mining from a hard disk', which isn't possible. You 'plot' on a hard-disk and that hard-disk is dead, and nothing will bring it back. You 'plot' on an SSD and that SSD is toast.

You run on NVME's and I'm finding 3-4 weeks and the NVME is running hot, and getting 'busy' count's from 'sudo nvme /dev/xxx'(linux), so far I find that most NVME's fail at about 25% of their TBW ( terabyte write warranty ), so sure send it back, and see if samsung will replace ( I'm there and so far none of come back, but in time I can see them calling this a non-warranty usage ). Remember these are 5-year drives, but if two weeks can use 25% of the TBW that means the puppy's are 'out of warranty' when their +2 months old. :(

Best results are running 256gb ram ( or 1TB if you can ), and setup a large ram disk, and config on lots of super fast machines to generate plots. 

The guys who wrote the 'plotting' tutorial say 'get data center' used NVME's off ebay, well those are long gone. Like everything else, just try to find anything these days, its like trying to find a GPU card +6gb.

Also IMHO 'plotting' has NOTHING to do with hard-disks, as they ain't even in the equation, I'm plotting using super fast NMVE's, and 24 core AMD top of the line with ton's of ram, totally computationally bound, nothing to do with 'drives', other than once a 108gb 'plot' is generated, then its written to a HDD.

But this isn't much different than 'bitcoin mining' where ASIC machines generate hashes to find one with N leading zero's, here 'win' is much the same, and the 'win' is 100% computationally bound, the only real difference is the history here is bitcoin was low-memory, and GPU took it  up to 6gb, and now CHIA takes it up to 256gb ( about that to make a plot ), but its all done in memory.

Lastly, what seems to die on NVME's IMHO is the controller, after 2-3 weeks they get hot, and start mal-functioning, of course the 'plotting sw' by 'chia' isn't designed to track, it just at the very end does this 'mis-match' test, which doesn't tell you anything, even worse is how plots will just 'hang' once the NVME starts failing ( hangs are always at funny numbers like 24.2gb, 42gb, 72gb ). Sure one could write monitoring tools in python, or bash, I'm sure eventually plotman will have to have a temp/kill switch enabled.

So at the end of the day, we're talking massive computation, and massive memory. I suspect that real soon we'll see 1TB (ram) mining (plotting boxes) coming out of China. Of course they'll be expensive, but if they can generate 100 plots/day, they can pay for themselves.

Of course what of the 'mythology' that CHIA is egalitarian, and lets people 'farm' with old boxes, .... sure I guess they can say "nobody saw this coming'? :)

At the end of the day, CHIA is just like BITCOIN the people with the biggest budgets for the newest hardware win, and nobody else, well except pre-miners :)

**Comment by SackofFluor** [comment]
I am not entirely sure, but is it possible that you haven't really understood the whole process?
Your actually winning chances come for the fully plotted Plots.
So if, for example, you filled up all your spare harddrives with 20TB of plots, then you can stop plotting and still have a chance to win, just leave the gui upon and the plots attached....

Buy some proper NVMEs with 1500-3600 TBW and you have enough headroom to plot between 700-1500 plots on this one alone, If your SSDs get hot... cool them? Stick a heatsink on or aim a fan on it?

**Reply by Harbourgroup** [reply]
When you stop plotting, your winning odd decreases as the total network grows. If you want to keep the same chance to win a Chia as yesterday, you have to keep plotting.

**Reply by SackofFluor** [reply]
Yes, but that is only possible when you have a constant supply with new HDDs...
Even when you don't add new storage, you still can have a little income on the side..

**Reply by room101-dev** [reply]
NO you can't I have dozens of NVME's, and some are rated 5,000 ( corsair ), but the problem is in big letters it says random r/w 500, the problem is 'plotting' is a random event, nada to do with your 5,000 sequential write/read biz, the problem is that plots need to be generated on dynamic-ram ( 1TB GDDR6 ) with 32 core controller, then you got something.

As to your BS about TBW, ... warranty is clear 600TBW for 1TB, I'm seeing NVME die at 150 or 25% of warranty, after that the controller on the NVME is toast, but it will still handle sequential, just not random

Lastly, this post was a rhetorical question about China making ASIC systems to generate plots, nothing about your spare hard drives problems

On average my NVME's are lasting 2 weeks before they die, I have fell back to chia-pos 1.3, as post that all the code on chia has bugs.

DEV refuses to even see, so China has no choice, since they're the real investors in CHIA to come out with plotting machines that don't kill drives.

Like I already said, bitcoin was 1MB, and gpu mining was 3 gb ( early eth ), now chia is 108gb; all that is happened is they keep raising the computation & storage barriers for difficulty, CHIA doesn't have a damn thing to do with hard-disk mining, HD's are just a place to store you 108gb lottery-ticket, for proof of creation.

**Reply by room101-dev** [reply]
> When you stop plotting, your winning odd decreases as the total network grows. If you want to keep the same chance to win a Chia as yesterday, you have to keep plotting.

U stop plotting and every 9 days the space doubles, today its 6.9EB, next week 12EB, that means your 200 plots today making one xch every two months, will generate 1 xch every 4 months next week

At 6EB all solo-mining is impossible, unless you have 10's of 1,000's plots, this is why morons solo mine, and smart people are on hpool earning xch all along while fools waited for baby-jeebuz to deliver the pool protocol

...

My post here was just about China making machines to deliver plots to fools who believed the hype. Our only hope now is the growth slows down, otherwise unless you have 10k plots you have no hope of making over $10/day once the chia-space hits 50EB

The problem right now is if I have to kill a $200 NVME to make 100 plots, my 500 plots cost me $1k; At 50EB then  my 500 plots generate less than $1/day, there is no point of powering the harvester, ( or running hpool ) time to pull the plug.

Yep, I know Bram's pooling protocol calls for a re-plot, where everybodys plots to date become worthless, I'm sure that will go over well in china where they have invested 10's of billions into CHIA IOUS, and HW; Bram/Hoffman have lost CHIA long ago, 70% is now controlled by HPOOL, and they **have already proved they cannot deliver working code,** both harvesting and plotting, in time if the main-net continues to be unstable I expect china to come out with their own chia-net that works.

**Reply by SackofFluor** [reply]
lol.... is there any factual basis to what you are saying?

How in the world do you kill your SSDs so fast?
My NVME is running for over 300Plots now and is still absolutely fine....

The network is 6,7EiB right now, I have around 67TB plotted and won a block on the Weekend....

The announced delay for the pool protocol happened  4 days or so ago and you blow it out of proportion because "they have already proved (proven?) they cannot deliver working code"

ffs pull the plug and take a chill pill......

from the data I can find hpool barely got 51% of the Netspace, which in theory allows for a 51% attack, but if I am not mistaken the time-component on the block chain should still prevent his....

I not saying you are entirely wrong, I only ask to tone your unfounded bullshit down a little bit.. that all....

**Comment by k3vst3r** [comment]
Whole design of chia is to avoid ASIC, didn't you read the memo about chia?

it's difficult to build ASIC for something storage bound, ASIC can't magically give you 1PB of storage

**Reply by Noob-granit** [reply]
I assume he means, attach your own storage and have an ASIC do the plotting to that storage. Using a massive RAM drive as a temp location.

**Reply by room101-dev** [reply]
> I assume he means, attach your own storage and have an ASIC do the plotting to that storage. Using a massive RAM drive as a temp location.

Thanks, clearly the reply person doesn't know what ASIC means, it just means ' applications custom chip', it could be anything

All that is needed is a 32-core cpu on a chip 1TB of dynamic GDDR-6 ram, and a couple NVME 2TB and I could be generating 100's of plots per hour, how much would somebody pay for the box?

**Reply by room101-dev** [reply]
> Whole design of chia is to avoid ASIC, didn't you read the memo about chia?
> 
> it's difficult to build ASIC for something storage bound, ASIC can't magically give you 1PB of storage

nobody needs 1PB of storage, we're talking  about making plots, that process only needs +300GB of dynamic ram, and later storage for 108gb files is a childs problem

The white/green chia papers are just pie in the sky design notes, that were written back in 2017 to get investors on board, nada to with anything called reality. Chia said it was going to be green too, we know that was a lie, or just ignorance???

ASIC APplication Specific Integrated Circuit,  it just means custom chip for a specific purpose, narrow market like creating chai plots.

**Reply by SackofFluor** [reply]
ok, then you can generate 100000x Chia plots a day.... how does that help you if you cant store them to actually farm?

**Reply by climba03003** [reply]
Yes, it can be generating 100 plots per hours, and your final destination (HDD) need to be fast enough to store them within an hours before your have next batch of plotting process.

That's means ASIC will only be usable for those who have tons of HDD (I think it needs more than hundred to compensate the slow transfer speed with an hour). That market was too small to make a ASIC.

**Comment by adamszen** [comment]
> I find that plotting is every bit as ASIC 'complex' as BTC mining, its the fastest CPUS with the most cores, and the largest ram disks ( 1TB on board DDR4 ) that make the fastest plots that win this game, how long before china has a CHIA box that spit's out 108GB plots? (100/day) I think very soon
[...]
> So at the end of the day, we're talking massive computation, and massive memory. I suspect that real soon we'll see 1TB (ram) mining (plotting boxes) coming out of China. Of course they'll be expensive, but if they can generate 100 plots/day, they can pay for themselves.
[...]

I'm using 10 cheap servers (below 200$ for each) only with SAS disks (not even one SSD!) and i'm plotting about 12TB (~120 plots) / day.

The key is multiplication (multiple servers), not the power of a single machine :)
